selinux build is broken with MRSV (1.79.0)

```
cargo +1.79.0 build --features=selinux
...
error[E0658]: lint reasons are experimental
   --> /home/drinkcat/.cargo/registry/src/index.crates.io-6f17d22bba15001f/selinux-0.5.0/src/errors.rs:144:24
    |
144 |     #[allow(dead_code, reason = "used by unit tests")]
    |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #54503 <https://github.com/rust-lang/rust/issues/54503> for more information

For more information about this error, try `rustc --explain E0658`.
error: could not compile `selinux` (lib) due to 1 previous error
```

2 options:
 - Revert 7bb2bb2f3054b8324d24f4ff24329032844b6661 to move selinux crate back from 0.5.0 to 0.4.6
 - Bump MRSV to 1.81.0

--- a/.github/workflows/CICD.yml
+++ b/.github/workflows/CICD.yml
@@ -11,7 +11,7 @@ env:
   PROJECT_NAME: coreutils
   PROJECT_DESC: "Core universal (cross-platform) utilities"
   PROJECT_AUTH: "uutils"
-  RUST_MIN_SRV: "1.79.0"
+  RUST_MIN_SRV: "1.82.0"
   # * style job configuration
   STYLE_FAIL_ON_FAULT: true ## (bool) fail the build if a style job contains a fault (error or warning); may be overridden on a per-job basis
 

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -16,7 +16,7 @@ repository = "https://github.com/uutils/coreutils"
 readme = "README.md"
 keywords = ["coreutils", "uutils", "cross-platform", "cli", "utility"]
 categories = ["command-line-utilities"]
-rust-version = "1.79.0"
+rust-version = "1.82.0"
 edition = "2021"
 
 build = "build.rs"

--- a/README.md
+++ b/README.md
@@ -14,7 +14,7 @@
 [![dependency status](https://deps.rs/repo/github/uutils/coreutils/status.svg)](https://deps.rs/repo/github/uutils/coreutils)
 
 [![CodeCov](https://codecov.io/gh/uutils/coreutils/branch/master/graph/badge.svg)](https://codecov.io/gh/uutils/coreutils)
-![MSRV](https://img.shields.io/badge/MSRV-1.79.0-brightgreen)
+![MSRV](https://img.shields.io/badge/MSRV-1.82.0-brightgreen)
 
 </div>
 
@@ -70,7 +70,7 @@ the [coreutils docs](https://github.com/uutils/uutils.github.io) repository.
 ### Rust Version
 
 uutils follows Rust's release channels and is tested against stable, beta and
-nightly. The current Minimum Supported Rust Version (MSRV) is `1.79.0`.
+nightly. The current Minimum Supported Rust Version (MSRV) is `1.82.0`.
 
 ## Building
 

--- a/tests/common/util.rs
+++ b/tests/common/util.rs
@@ -408,7 +408,7 @@ impl CmdResult {
 
     /// Returns whether the program succeeded
     pub fn succeeded(&self) -> bool {
-        self.exit_status.map_or(true, |e| e.success())
+        self.exit_status.is_none_or(|e| e.success())
     }
 
     /// asserts that the command resulted in a success (zero) status code

